appyx-components/demos/common/build.gradle.kts

```diff
@@ -6,6 +6,10 @@ plugins {
     id("appyx-publish-multiplatform")
 }
 
+publishingPlugin {
```

Comment:
Why publish demos?

Reply:
we want to showcase some stuff but we won't commit to supporting it